web3/content_script.js

```diff
@@ -155,23 +155,41 @@ function main () {
     // lazy load styles
     const styles = require('./styles.css');
     const iframe = document.createElement('iframe');
-    iframe.className = styles.iframe__main;
+    iframe.className = [ styles.iframe__main, styles.iframe__open ].join(' ');
     iframe.src = chrome.extension.getURL('web3/embed.html');
     iframeInjected = iframe;
 
     window.addEventListener('message', (ev) => {
       if (ev.source !== iframe.contentWindow) {
         return;
       }
-      if (!ev.data.type || ev.data.type !== EV_SIGNER_BAR) {
+
+      if (!ev.data.type) {
         return;
       }
-      if (ev.data.opened) {
-        iframe.classList.add(styles.iframe__open);
-      } else {
-        iframe.classList.remove(styles.iframe__open);
+
+      switch (ev.data.type) {
+        case EV_SIGNER_BAR:
+          if (ev.data.opened) {
+            iframe.classList.add(styles.iframe__open);
+          } else {
+            iframe.classList.remove(styles.iframe__open);
+          }
+
+          break;
+
+        case EV_IFRAME_STYLE:
```

Comment:
I think it would be better to give the event more specific name, since it's also closes the iframe and is fired after the resize happened (it's not simply updating the styles as the name suggests now).